frontend/public/components/dashboards-page/overview-dashboard/details-card.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { connect } from 'react-redux';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../dashboard/dashboard-card';
+import { DetailsBody, DetailItem } from '../../dashboard/details-card';
+import { withDashboardResources, DashboardItemProps, FirehoseResource } from '../with-dashboard-resources';
+import { InfrastructureModel } from '../../../models';
+import { referenceForModel } from '../../../module/k8s';
+import { FLAGS } from '../../../const';
+import { featureReducerName, flagPending } from '../../../reducers/features';
+import { RootState } from '../../../redux';
+
+const OPENSHIFT_VERSION_QUERY = 'openshift_build_info{job="apiserver"}';
```

Comment:
Any reason not to use the `ClusterVersion` resource.

We should try to show consistent versions everywhere in the console, and other places we use `ClusterVersion`. It will also let us use a true watch instead of polling.

Reply:
nice! Using `ClusterVersion` now